Web/starter_project_vanguard/components/team/Member.tsx

```diff
@@ -0,0 +1,36 @@
+import Image from "next/image"
+import Link from "next/link"
+
+interface MemberProps {
+    name?: string,
+    picture?: string,
+    role?: string,
+    description?: string,
+    links?: string[]
+}
+const Member = ( {name = "Yared Tegegn", picture="https://media.licdn.com/dms/image/C4E03AQEVlrjCsy97_Q/profile-displayphoto-shrink_800_800/0/1661101696311?e=2147483647&v=beta&t=JK2LQ6ZoiQeWxoLnrPUwUiX5Bqi5mHBTIVKh4HK0cQk", role="Intern", description="Yared is a motivated software engineer looking to pursue a successful career in software development where he can help deliver software solutions to social problems. His skills include Web Development, Data Structures, Machine Learning, and DevOps. Natnael’s hobbies include coding, playing soccer, and watching movies.", links=["https://github.com/", "https://github.com/", "https://github.com/"]}: MemberProps) => {
```

Comment:
1. can you use some syntax like React.FC or sth to define the component here (took me a while to recognize that it is a component)
2. please don't set the defaults here (i don't think the component needs defaults in the first place) - I recommend you pass your default member object from where you are rendering it multiple times if you have to
3. If possible try to find a better place to define the data and the Member type as it might be a type that could be useful in multiple components (or maybe not idk, but at least the data stored here within the code just creates confusion) - you might even use a JSON to store the mock data and see how you would render that
4. might be better to rename it as `MemberCard` as it better describes what it would look like (also remember to update the prop interface accordingly too)